src/ocm/components/clusterConfiguration/ReviewCluster.tsx

```diff
@@ -55,7 +55,7 @@ const ReviewCluster: React.FC<{ cluster: Cluster }> = ({ cluster }) => {
       />
       <DetailItem
         title={'Stack type'}
-        value={isIPv4(cluster) ? 'IPv4' : 'Dual-stack'}
+        value={isDualStack(cluster) ? 'Dual-stack' : 'IPv4'}
```

Comment:
FYI, that the day they'll implement single-stack w/IPv6 only, we'll need to change this...

Reply:
We will need to change a lot of things once that day comes :slightly_smiling_face: